### DISCUSSION TITLE
How to preserve cwd of nvim?

### DISCUSSION THREAD

**Discussion by zpah** [nvim-neo-tree/neo-tree.nvim#463] [Q&A] [answered]
Command Neotree changes cwd unexpectedly.
Preconditions:
1. Current working directory is /tmp/d1
2. Opened two files from different directories:
```
:ls
#    f1
%a   /tmp/d2/f2
```
Current buffer is /tmp/d2/f2
Command `:Neotree float toggle reveal reveal_force_cwd` unconditionally changes cwd.
Command `:Neotree float reveal toggle` annoys by confirmation, but, nevertheless, damages my cwd.

How to keep cwd unchanged by Neotree ?

**Comment by cseickel** [comment, marked as answer]
The short answer is:

```lua
    require("neo-tree").setup({
      filesystem = {
        bind_to_cwd = false
      }
    })
```

See `:h neo-tree-cwd` for full details about cwd.

**Comment by zpah** [comment]
Thanks for your recipe!
It works as i expected with command: `:Neotree float toggle reveal reveal_force_cwd`
I should read the documentation more carefully :)
Thank you for the excellent plugin!
